frontend/src/app/modules/auth/login/login.component.ts

```diff
@@ -0,0 +1,95 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
+import {Router} from "@angular/router";
+import {AuthenticationService} from "../../shared/services/authentication.service";
+import {DataService} from "../../shared/data/data.service";
+import {ToastrService} from "ngx-toastr";
+import { DynamicFormComponent } from '../../shared/components/dynamic-form/dynamic-form.component';
+import { LocalStorageService } from 'src/app/core/services/local-storage/local-storage.service';
+import { localKeys } from 'src/app/core/constants/localStorage.keys';
+import { map } from 'rxjs';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  @ViewChild('loginForm') loginForm!: DynamicFormComponent;
+  rememberMe:boolean = false;
+  controls:any = {
+    controls: [
+      {
+        name: 'email',
+        label: 'Email Id',
+        value: '',
+        type: 'email',
+        placeHolder: 'abc@example.com',
+        errorMessage: 'Please enter registered email id',
+        validators: {
+          required: true,
+          pattern: '[A-Za-z0-9._%-]+@[A-Za-z0-9._%-]+\\.[a-z]{2,3}'
+        },
+      },
+      {
+        name: 'password',
+        label: 'Password',
+        value: '',
+        type: 'password',
+        placeHolder: 'Enter password',
+        errorMessage: 'Please enter your registered password',
+        validators: {
+          required: true,
+          minLength: 8,
+          pattern: "^[a-zA-Z0-9!@#%$&~*^()\\-`.+,/\"]*$"
+        },
+      },
+    ]
+  };
```

Comment:
Can we think of dynamic forms coming from mongo? rather than hard coding it?

@Apoorvgarg-creator